### PR TITLE
[minor] Export InteractionManager

### LINKED ISSUE #49: InteractionManager.createInteractionHandle is not a function
The minimal mocha test suite to reproduce this problem is following:

``` js
// tests/animatedTest.js
import {Animated} from 'react-native'

describe('Animated', () => {
  it('should animate', () => {
    const value = new Animated.Value(0)
    Animated.timing(
      value,
      {
        toValue: 1,
        delay: 1000
      }
    ).start()
  })
})
```

When running with:

```
mocha --require react-native-mock/mock --compilers js:babel-core/register tests/animatedTest.js       
```

I get the following error:

```
     TypeError: _InteractionManager2.default.createInteractionHandle is not a function
      at AnimatedValue.animate (node_modules/react-native-mock/build/api/Animated/AnimatedImplementation.js:750:52)
      at Object.start (node_modules/react-native-mock/build/api/Animated/AnimatedImplementation.js:1543:23)
      at Context.<anonymous> (animatedTest.js:12:7)
```

It looks like [InteractionManager.js](https://github.com/lelandrichardson/react-native-mock/blob/6e1c585916aeca7063a4e024a348c433aa0fcad8/src/api/InteractionManager.js) is missing `module.exports` statement.

### PR DESCRIPTION
**Pull request by tomkur**
Fixes #49

### CODE REVIEW

**Comment by RealOrangeOne** [comment]
Looks good to me. Shall merge when tests have passed!

**Comment by tomkur** [comment]
Problems with running node v4, unrelated to this PR

**Comment by RealOrangeOne** [comment]
Agreed. Shall remove Nodev4 from the project in this case. It's considered out of date anyway. Merging

**Comment by tomkur** [comment]
Thanks!
When could I expect publishing it to NPM?
Or maybe a more general question: how can I make use of this merged change immediately? If I change dependency to github repo instead of npm module, the prepublish scripts are not run, so it doesn't work. If I run prepublish script manually, I get some errors from babel.
Is there an easy way to do that?

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "A fully mocked and test-friendly version of react native",
   "main": "build/react-native.js",
   "scripts": {
-    "prepublish": "npm run build",
+    "postinstall": "npm run build",
```
Comment (jhen0409) [inline package.json:7]: This is wrong...

**Comment by tomkur** [comment]
Oops, that was accidental...

**Comment by RealOrangeOne** [comment]
Ah damn, I missed that. Shall add that back now manually myself

**Pull request merged**